src/media/js/main.js

```diff
@@ -160,8 +162,10 @@ function(_) {
 
 
             var manifest_url = settings.manifest_url;
-            // Note: only the iframed app defines a manifestURL for now.
-            if (manifest_url) {
+            // We want to check for an update, so we need the manifest url. In
+            // addition, we only want to show the update notification banner if
+            // we are inside an app (not just browsing the website).
+            if (manifest_url && (is_packaged_app || is_iframed_app)) {
```

Comment:
Ultimately, when we switch to a true packaged app, it'll reuse the same manifest url as the iframed app, since we can't change manifest URL for a given app. But we haven't switched yet - we are still using the iframed app for now, and we don't want people testing the true packaged app to override their app with the iframed one. That's why I left it commented out in `settings_local_package.js` for now.

I'll update the comment.